auth_password_settings/data/auth_password_settings_data.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
```

Comment:
It could be great to set an noupdate option here. Otherwise, the changes of the settings will be erased at each update.